src/main/kotlin/no/nav/syfo/melding/database/MeldingQuery.kt

```diff
@@ -76,17 +76,18 @@ const val queryGetMeldingerForConversationRefAndArbeidstakerident =
     """
         SELECT *
         FROM MELDING
-        WHERE conversation_ref = ? AND arbeidstaker_personident = ? AND NOT innkommende
+        WHERE (uuid = ? OR conversation_ref = ?) AND arbeidstaker_personident = ? AND NOT innkommende
```

Comment:
Litt rubberducking for meg selv her: Her vil jo et treff på `conversation_ref` gi potensielt flere meldinger i lista (feks original melding og påminnelse), mens treff på `uuid` vil gi én melding. Det er vel uansett ikke så farlig, ettersom det eneste vi bryr oss om er hvorvidt det er et treff på utgående melding i vår DB, og at den utgående meldingen sin `conversationRef` blir brukt til å lagre ny innkommende melding 👌🏼 
Enig?

Reply:
Ja, enig i det.

Alle steder der dette query'et brukes så hentes det ut en liste, men så bruker man bare første element i lista (dvs den opprinnelige meldingen sendt fra oss).